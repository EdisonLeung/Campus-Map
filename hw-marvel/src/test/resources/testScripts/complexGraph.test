# tests pathfinding on a complex graph with many nodes and edges. This tests will
# test multiple path findings to test if pathfinding works on a complex graph

# Load graph
LoadGraph complexGraph complexGraph.csv

# path finding
FindPath complexGraph a d

FindPath complexGraph a b

FindPath complexGraph c b

FindPath complexGraph d b

FindPath complexGraph b d