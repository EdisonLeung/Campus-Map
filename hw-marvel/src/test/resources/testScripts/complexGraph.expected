# tests pathfinding on a complex graph with many nodes and edges. This tests will
# test multiple path findings to test if pathfinding works on a complex graph

# Load graph
loaded graph complexGraph

# path finding
path from a to d:
a to d via AD

path from a to b:
a to b via ABa

path from c to b:
c to a via AC
a to b via ABa

path from d to b:
d to a via AD
a to b via ABa

path from b to d:
b to a via ABa
a to d via AD